app/utils/zip-agenda-files.js

```diff
@@ -7,14 +7,14 @@ function registerJobToStore(job, store) {
 }
 
 function prettifyAgendaName(agenda) {
-  if (agenda.get('isDesignAgenda')) {
+  if (agenda.isDesignAgenda) { // async computed!
```

Comment:
De computed zelf is niet async, er wordt wel vanuit gegaan dat agenda.status reeds is geladen als je die computed aanroept (anders bestaat er ook een async method `asyncCheckIfDesignAgenda`)
In de view waar je de agenda kan printen, is "status" al included in de query.

Reply:
De comment hierbij, dat voegt op zich niet veel bij en is wat onduidelijk.
is dit een todo? een waarschuwing ?
Als async een probleem zou zijn, waarom maken we de function dan ook niet async en werken we met await ?